lib/logstash/codecs/cef.rb

```diff
@@ -114,6 +117,9 @@ def decode(data)
     end
 
     yield event
+  rescue => e
+    @logger.error("Failed to decode event payload. Generating failure event with payload in message field.", :error => e.message, :backtrace => e.backtrace, :data => data)
+    yield LogStash::Event.new("message" => data, "tags" => ["_cefparsefailure"])
```

Comment:
logstash-filter-json allows to configure the tag for the failure case: https://github.com/logstash-plugins/logstash-filter-json/blob/master/lib/logstash/filters/json.rb#L60
should we provide similar behavior?